mm2src/lp_swap.rs

```diff
@@ -280,29 +278,7 @@ lazy_static! {
     /// Default atomic swap payment locktime, in seconds.
     /// Maker sends payment with LOCKTIME * 2
     /// Taker sends payment with LOCKTIME
-    static ref PAYMENT_LOCKTIME: u64 = get_payment_locktime_from_mm2config();
-}
-
-#[cfg(feature = "custom-swap-locktime")]
-/// Reads `payment_locktime` from cli or MM2.json configuration and returns it.
-/// Returns 900 when `payment_locktime` is invalid or not provided.
-fn get_payment_locktime_from_mm2config() -> u64 {
-    let args: Vec<OsString> = env::args_os().collect();
-    let first_arg = args.get(1).and_then(|arg| arg.to_str());
-    let conf = super::get_mm2config(first_arg).unwrap();
-
-    match conf["payment_locktime"].as_u64() {
-        Some(lt) => lt,
-        None => {
-            let def = 900;
-            warn!(
-                "payment_locktime is either invalid type or not provided in the configuration or
-                MM2.json file. payment_locktime will be proceeded as {} seconds.",
-                def
-            );
-            def
-        },
-    }
+    pub(crate) static ref PAYMENT_LOCKTIME: Mutex<u64> = Mutex::new(0);
```

Comment:
I think the `PAYMENT_LOCKTIME` type should be `Arc<AtomicU64>`. Also, the default value of `900` should be assigned here straight away.

Reply:
I'd rather make it `AtomicU64` without `Arc` and `lazy_static` :)
```rust
static PAYMENT_LOCKTIME: AtomicU64 = AtomicU64::new(CUSTOM_PAYMENT_LOCKTIME_DEFAULT);
```

Reply:
> 

Fixed this problem with providing default value as const in mm2.rs.

Reply:
> > I think the `PAYMENT_LOCKTIME` type should be `Arc<AtomicU64>`. Also, the default value of `900` should be assigned here straight away.
> 
> > I think the `PAYMENT_LOCKTIME` type should be `Arc<AtomicU64>`. Also, the default value of `900` should be assigned here straight away.
> 
> I thought it would be better to keep all the data controll of this variable in mm2.rs so I kept it as 0. Anyway, I updated it as 900 now, but I am still not sure if it's a good idea. For me it looks kind a duplicated value. When someone wants to update default value, probably he/she will get confused at the beginning and then update both of them :)

Fixed this problem with providing default value as const in mm2.rs.